Gemfile

```diff
@@ -55,6 +55,8 @@ gem 'surveyor', :git => 'https://github.com/caboteria/surveyor.git', branch: 'ra
 gem 'therubyracer', '0.10.2', :platforms => :ruby
 gem 'uglifier', '>= 1.0.3'
 gem 'will_paginate'
+gem 'omniauth-cas'
```

Comment:
Great thanks for reviewing it and having patience with us! Sorry for the confusion on the pull requests, we're just getting started on sending changes your way.